mne_qt_browser/_pg_figure.py

```diff
@@ -2598,17 +2598,26 @@ class PyQtGraphBrowser(BrowserBase, QMainWindow, metaclass=_PGMetaClass):
 
     gotClosed = pyqtSignal()
 
-    def __init__(self, splash=None, **kwargs):
+    def __init__(self, **kwargs):
         self.backend_name = 'pyqtgraph'
 
         BrowserBase.__init__(self, **kwargs)
         QMainWindow.__init__(self)
-        self._splash = splash
```

Comment:
@marsipu I didn't realize that the `kwargs` ended up being in `self.mne.<kwargs>`. I simplified a bit of the code here based on this.

Reply:
Nice:+1: